playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders/terraform.tf

```diff
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.40.0"
+      version = "~> 4.62.0"
```

Comment:
Do we want to update version in other TF scripts too? (not in this PR)

Reply:
This is because i needed latest to support `script` field for triggers. Usually I don't recommend to go with latest unless necessary. We should have separate discussion and task for this